### PR TITLE
Add DCT to eidos and to JLD

### PR DESCRIPTION
**Pull request by kwalcock**
This is still in the final stages of testing, but people can look.

### CODE REVIEW

**Comment by kwalcock** [comment]
useTimeNorm is usually not true, so I'm inclined to be lenient and merge this.  We don't have a solution implemented to deal with reading the model from a jar.  For this large run, I've just got a hardcoded patch.

File: src/main/scala/org/clulab/wm/eidos/apps/ExtractJsonLDFromFile.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ object ExtractJsonLDFromFile extends App {
   val pw = FileUtils.printWriterFromFile(outputFile)
   // 3. Extract causal mentions from the text
   val annotatedDocument = Timer.time("Just extraction") {
-    reader.extractFromText(text)
+    reader.extractFromText(text, documentCreationTime = Some("2018-09-07"))
```
Comment (MihaiSurdeanu) [inline src/main/scala/org/clulab/wm/eidos/apps/ExtractJsonLDFromFile.scala]: What is this arbitrary date?

Comment (kwalcock) [reply src/main/scala/org/clulab/wm/eidos/apps/ExtractJsonLDFromFile.scala]: Thanks.  Leftover from testing.  Removing...

**Comment by kwalcock** [comment]
Further changes in master then.

**Comment by EgoLaparra** [comment]
I have tried to run the last changes and everything works as expected.

**Pull request merged**